picard/coverart/__init__.py

```diff
@@ -72,10 +73,10 @@ def retrieve(self):
         else:
             log.debug("Cover art disabled by user options.")
 
-    def _set_metadata(self, coverartimage, data):
+    def _set_metadata(self, coverartimage, data, info):
```

Comment:
Maybe it would be a bit clear to use `image_info` instead of more generic `info`.
Same for all occurrences below